source/assets/stylesheets/base/_typography-font-face.scss

```diff
@@ -0,0 +1,32 @@
+@font-face {
+  font-family: "Big Caslon";
+  src: url("../fonts/BigCaslon.eot"); /* IE9 Compat Modes */
+  src: url("../fonts/BigCaslon.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */
+       url("../fonts/BigCaslon.woff") format("woff"), /* Modern Browsers */
+       url("../fonts/BigCaslon.ttf")  format("truetype"), /* Safari, Android, iOS */
+       url("../fonts/BigCaslon.svg#b27b23f39781b88bedc6e519ef754c9f") format("svg"); /* Legacy iOS */
+  font-style: normal;
+  font-weight: 400;
+}
+@font-face {
+  font-family: "Input";
+  src: url("../fonts/Input-Regular_(InputMono-Light).eot"); /* IE9 Compat Modes */
```

Comment:
This naming is confusing `Regular...Light`. Not a big deal, but just thought I'd point it out.